src/main/features/ccj/views/their-details.njk

```diff
@@ -1,70 +0,0 @@
-{% extends "layout.njk" %}
```

Comment:
@singh-surjeet do you know when / why related route file was removed?

Reply:
I spoke @timja-kainos to double check. We couldn't find related route and assumed we forgot to remove view.